HTML/EN/settings/server/basic.html

```diff
@@ -89,6 +89,12 @@
 		[% END %]
 	[% END %]
 
+	[% WRAPPER settingSection %]
```

Comment:
Please move this out of basic settings to My Music instead.

Reply:
Moved $skipsentinel into `folderFilter()` sub.

I'm not sure if My Music is the right place for the skipsentinel setting. My Music deals with music (meta)data content. Skipsentinel operates on the scanning level, not the content level.

Reply:
It's a better place than Basic Settings, as it's an advanced setting. Please :-).

Reply:
Ok, done.